windows/flutter/generated_plugin_registrant.cc

```diff
@@ -0,0 +1,15 @@
+//
```

Comment:
What's with all these windows related files?

We're not creating a windows application, it's polluting the codebase